Jenkinsfile

```diff
@@ -132,6 +132,10 @@ def cancel_previous_build() {
 }
 
 def should_skip_ci(pr_number) {
+  if (env.BRANCH_NAME == 'main' || env.BRANCH_NAME == 'ci-docker-staging') {
```

Comment:
it seems like we should never skip ci when doing any branch build, no? can we test for that instead?

Reply:
good point, updated to check for branches instead